src/common/helpers/ImportHelpers.ts

```diff
@@ -12,5 +12,17 @@ export function getImportInsertPosition(editor: TextEditor | undefined): Positio
     if (!editor) {
         return new Position(0, 0);
     }
-    return editor.document.lineAt(0).text.match(/use strict/) ? new Position(1, 0) : new Position(0, 0);
+    if (editor.document.lineAt(0).text.trim().match(/^\/\//)) {
```

Comment:
The basic idea is nice :-)
I'd refactor the logic though..

Basically as long as one of the given expressions is true, you could increment the line number

I'd do:
```typescript
const lines = editor.document.getText().split('\n');
const index = lines.findIndex(line => !line.match(/^\s*(\/\/|\/\*\*|\*\/|\*|['"]use strict['"])/g));
return new Position(Math.max(0, index), 0);
```

Reply:
If you want to, you can actually open an issue for this, and I'll paste my code that I just wrote with a test in there. So we have a proper issue with a pull request. If you want.

Or you use the code or any other refactoring and write some tests for it ;-)